stdlib/2.7/calendar.pyi

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Iterable, Optional, Tuple
+import datetime
+
+YearType = int
+MonthType = int
+DayType = int
+WeekdayType = int
```

Comment:
In general I want to resist seeing the type stubs as yet another form of docs. It should serve one need: to provide stubs usable by automatic type checkers. Also the majority of these are redundant, e.g. `month: MonthType` repeated over and over (in fact `MonthType` is *less* helpful than `int`!).